### PR TITLE
modify tests relate to get uploads and add pagination test as well

### PR DESCRIPTION
**Pull request by zaclinsage**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,14 +49,20 @@
         <dependency>
             <groupId>org.sagebionetworks.bridge</groupId>
             <artifactId>rest-client</artifactId>
-            <version>0.12.23</version>
```
Comment (alxdarksage) [inline pom.xml]: bump to 0.12.26

File: pom.xml [hunk 2]
```diff
@@ -49,14 +49,20 @@
...
             <groupId>org.sagebionetworks</groupId>
             <artifactId>synapseJavaClient</artifactId>
             <version>161.0-4-g843de2c</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.21</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (alxdarksage) [inline pom.xml]: This is included in the rest client, is that not included as part of the tests? I think Joshua just removed logging dependencies from these tests

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyTest.java [hunk 3]
```diff
@@ -375,13 +388,38 @@ public void getStudyUploads() throws Exception {
...
             assertEquals(endTime, results.getEndTime());
 
             assertEquals(count+2, results.getItems().size());
             assertNotNull(getUpload(results, uploadSession.getId()));
             assertNotNull(getUpload(results, uploadSession2.getId()));
+
+            // then test pagination by setting max pagesize to 1
+            UploadList pagedResults = studiesApi.getUploads(startTime, endTime, 1L, null).execute().body();
+            assertEquals(startTime, pagedResults.getStartTime());
+            assertEquals(endTime, pagedResults.getEndTime());
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyTest.java]: Add test for page size to be null... this is the test that will verify the BridgePF routes file allows the parameter to be optional...

**Review by alxdarksage — CHANGES_REQUESTED** [review]
A couple of minor changes. Also, do we have a test to verify that an admin can logically delete a study, then it doesn't appear in the APIs, then the admin developer can re-enable it? As much of that logic that is in the API, we should have tests for.
